clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -923,7 +930,7 @@ private void handleApiVersionsResponse(List<ClientResponse> responses,
     private void handleDisconnections(List<ClientResponse> responses, long now) {
         for (Map.Entry<String, ChannelState> entry : this.selector.disconnected().entrySet()) {
             String node = entry.getKey();
-            log.debug("Node {} disconnected.", node);
+            log.info("Node {} disconnected.", node);
```

Comment:
It makes me thinking: why not consolidating `disconnect` to call `processDisconnections` but have two call paths here?

Reply:
I agree there is room for consolidation, but I was trying to avoid refactoring here. Could we consider this separately?